component/block_cache/stream.go

```diff
@@ -41,14 +41,12 @@ import (
 	"github.com/Azure/azure-storage-fuse/v2/common/config"
 	"github.com/Azure/azure-storage-fuse/v2/common/log"
 	"github.com/Azure/azure-storage-fuse/v2/internal"
-	"github.com/Azure/azure-storage-fuse/v2/internal/handlemap"
 
 	"github.com/pbnjay/memory"
 )
 
 type Stream struct {
 	internal.BaseComponent
```

Comment:
We can remove this and other methods like SetNextComponent(), Priority(), etc. if not needed.